8.0 branch

mileo opened the pull request:
Hello,

A few months ago I created a "PR" for the creation of a branch 8.0 for this repository. 

> Can someone that have push permission to upstream, create the 8.0 branch? https://github.com/OCA/l10n-brazil/pull/109

Now this is a pull request of months of work from @danimaribeiro @mileo @mstuttgart @Mackilem to migrate l10n_br to 8.0. Here is the currenty WIP:
1. Make all modules installable; OK
2. Fix bugs and make then operational; OK
3. Migrate to new API and flake8 pass:
- [X] l10n_br_account
- [ ] l10n_br_account_payment
- [x] l10n_br_account_product
- [ ] l10n_br_account_product_service
- [ ] l10n_br_account_service
- [ ] l10n_br_account_voucher
- [x] l10n_br_base
- [x] l10n_br_crm
- [ ] l10n_br_crm_zip
- [x] l10n_br_data_account
- [x] l10n_br_data_account_product
- [ ] l10n_br_data_account_service
- [x] l10n_br_data_base
- [x] l10n_br_delivery
- [ ] l10n_br_hr_timesheet_invoice
- [x] l10n_br_product
- [x] l10n_br_purchase
- [x] l10n_br_sale
- [x] l10n_br_sale_product
- [ ] l10n_br_sale_service
- [x] l10n_br_sale_stock
- [x] l10n_br_stock
- [ ] l10n_br_zip

During migration we also did everything possible to improve functionality. There have been improvements in crm ( company and contact creating on lead ) and account ( full operational MD-E ). If anyone wants to help or is already working on migration we will be happy to know.

In https://github.com/OCA/l10n-brazil/pull/109#issuecomment-64740403 and as told before yesterday on skype (with @danimaribeiro): @renatonlima informed us that he had worked on migration in November and publish your code today. For some unknown reason the branch 8.0 has not already been created and we had no feed back. So that's the PR of our work.

But both KMEE as other contributors are waiting a long time to be able to contribute more actively and without a very large delay. We would like to request that someone looked fondly our actions so that we can continue to contribute more and better. Thus, we would not have only one company handling the official repository of the Brazilian location. We can contribute to reviewing and analyzing third-party code task to be able to streamline the approvals in our location.

The intention of this request is to contribute to the development of new features and fixes in odoo environment. In KMEE we always seek to promote what we are working from the first commit. During this one year that we are official partner, there was no lack of good intention, and let's say that it was not the easiest year to get into the game and we hadn't any malfeasance.

Maybe it's my point of view, to want things to evolve too fast, anyway, I'm sorry if there was any misunderstanding

 Best regards

danimaribeiro committed:
Corrigido erro na tag de um field na view Cancelar Documento Eletrônico

danimaribeiro committed:
Fixed imports for version8.0

danimaribeiro committed:
Corrigindo imports para versão 8.0

danimaribeiro committed:
Migração de módulos para versão 8.0

danimaribeiro committed:
Migraçao v8 - removido referencias a stock.picking.in out e sale.shop

danimaribeiro committed:
Adicionado ordem aos documentos de eventos para mostrar recentes por primeiro

danimaribeiro committed:
Melhorado layout do botão buscar cep

danimaribeiro committed:
Removido totalmente referencia ao shop_id, locais onde usava shop_id para pegar company_id foi modificado por user_id

sadamo committed:
[NEW] Rateio de custos na invoice

mileo committed:
[REF][BUG] Fix issue #112, by adding costs calc in account.tax

mileo committed:
[FIX] Finish invoice costs ratio and accounting of them.

Conflicts:
	l10n_br_account_product/account_invoice.py

mileo committed:
[REM] Remove the old method of calculation

mileo committed:
[FIX] Fixed issue #99

mileo committed:
[FIX][REF] Fix issue, change class name, flake8 pass

mileo committed:
[FIX] Issue TypeError: object of type 'int' has no len()

mileo committed:
[FIX] Line id

mileo committed:
[FIX] fixed #115

danimaribeiro committed:
[FIX] fixes #115

danimaribeiro committed:
[FIX] Calc tax_discount on l10n_br_sale/sale_product

danimaribeiro committed:
Merge remote-tracking branch 'fork/develop' into feature/migracao_8.0

Conflicts:
	l10n_br_account/l10n_br_account_view.xml

danimaribeiro committed:
Fixed demo product data

danimaribeiro committed:
Changed reference from account_voucher to account

danimaribeiro committed:
Changed paths from travis build

danimaribeiro committed:
Updated readme

danimaribeiro committed:
Fixing travis build

danimaribeiro committed:
Upgraded method compute_all

danimaribeiro committed:
Fixed a few number of bugs that were preventing stock to install

danimaribeiro committed:
Merge commit 'refs/pull/102/head' of https://github.com/OCA/l10n-brazil into develop

danimaribeiro committed:
Merge PR '#113' of https://github.com/OCA/l10n-brazil into develop

danimaribeiro committed:
Merge pull request '#116' of https://github.com/OCA/l10n-brazil into develop

danimaribeiro committed:
Merge branch 'fork/feature/migracao_8.0' into develop

Conflicts:
	l10n_br_account_product/l10n_br_account_product.py
	l10n_br_stock/stock.py

danimaribeiro committed:
Fixed problems with merge

danimaribeiro committed:
Fixed CFOP form view

danimaribeiro committed:
Fixed a bunch of views to display the correct label

danimaribeiro committed:
Fixing problems

danimaribeiro committed:
Fixed problems in Validate invoice

danimaribeiro committed:
Fix IE problem and product_id_change issues

danimaribeiro committed:
Improving fiscal rules view

mileo committed:
Update .travis.yml

danimaribeiro committed:
Changed test files path to make modules instalable

danimaribeiro committed:
Fixed errors when creating a new purchase order

danimaribeiro committed:
Fixed errors in fiscal position template

danimaribeiro committed:
Getting the correct account-fiscal-rule

danimaribeiro committed:
Fixing fiscal position change

danimaribeiro committed:
Fixed problem with loading company taxes

Mackilem committed:
ajustado o nome da page

Mackilem committed:
inseridos os campos fiscais na classe product.template

Mackilem committed:
corrigida a view para que apareceçam os campos

danimaribeiro committed:
Revertendo alterações do product.product

danimaribeiro committed:
Fixed product views

danimaribeiro committed:
Fix issue #2 - Correct ident

danimaribeiro committed:
Fixing service code to work with nfe 3.10

danimaribeiro committed:
Added id_dest to demo data and to fiscal.position.template

danimaribeiro committed:
Moved id_dest field to be copied from template

danimaribeiro committed:
Setting default value to freight_responsibility to fix build

mileo committed:
[MIG] Migrate l10n_br_base to new api v8 / flake8 pass

mileo committed:
[MIG] Migrate l10n_br_base to new api v8 / flake8 pass

danimaribeiro committed:
Merge from Fixed unit tests in validate_ie_param

danimaribeiro committed:
Enabling unit tests travis script

danimaribeiro committed:
Fixed validate IE for TO state

mileo committed:
[MIG] Migrate l10n_br_product to v8

mileo committed:
[MIG] Migrate l10n_br_product to v8

mileo committed:
[MIG] Migracao do modulo l10n_br_data_base para 8.0

mileo committed:
[MIG] Migracao do modulo l10n_br_data_base para 8.0

mileo committed:
[MIG] Migrate l10n_br_crm to new api v8

mileo committed:
[MIG] Migrate l10n_br_crm to new api v8

mstuttgart committed:
[MIG] Import e fields zatualizados para v8
[UPDATE] Atualização copyrigth

Mackilem committed:
corrigida a view para que a NCM nem o TIPO DE SERVIÇO não sejam obrigatorios quando for cadastrado um serviço.

Mackilem committed:
Merge branch 'migracao_8.0' of https://github.com/Trust-Code/l10n_br_core into migracao_8.0

mstuttgart committed:
[MIG] Migração parcial doa métodos onchange
[FIX] Bug ao abrir a view para adicionar produtos ne aba "Transferência"
[FIX] Bugao chamar o método onchange_partner_in do super

mstuttgart committed:
[MIG] Módulo l10n_br_stock migrado parcialmente para V8.
Ainda resta migrar as classes pertencentes ao wizard do módulo.

danimaribeiro committed:
Adicionado nova opção aos documentos

mstuttgart committed:
[FIX] Erro do método onchange_partner_in ao chamar o super da classe base. Retirado chamada do super de dentro do metodo.
O erro indicava que a classe base da classe stock_picking do módulo account_fiscal_position_rule_stock não possuia
o método on_change_partner_in.

mstuttgart committed:
[MIG] Onchange_partner_in migrado para v8. Foi preciso realizar do repositório da Alkretion
no módulo oca/account-fiscal-rule/account-fiscal-position-rule-stock

mstuttgart committed:
Ajustes no codigo

mileo committed:
[FIX] Wrong decorator, fix on_change partner_id on sale order

mileo committed:
Merge branch '8.0' into HEAD

Conflicts:
	l10n_br_base/res_partner.py

mileo committed:
Update README.md

mileo committed:
Update README.md

mileo committed:
Update README.md

mileo committed:
Update README.md

mstuttgart committed:
Limpeza do código

mstuttgart committed:
[MIG] imports e fields das classes da pasta wizard migrados para v8

mstuttgart committed:
[MIG] imports atualizados

mileo committed:
[FIX] Crm Version, On change partner id in sale order.

mileo committed:
[FIX] Crm Version, On change partner id in sale order.

mileo committed:
Merge pull request #4 from kmee/8.0

[FIX] Crm Version, On change partner id in sale order.

mstuttgart committed:
Ajuste chamada do super

mstuttgart committed:
data remidos para teste

mstuttgart committed:
[UPDATE] Conversão fr 'quantity' mudou para float.

mstuttgart committed:
Limpeza do codigo

mstuttgart committed:
[FIX] Corrigido bug do erro: TypeError: can't multiply sequence by non-int of type 'float'.
 O erro ocorre durante a execução do metodo 'compute_all' que se encontra dentro do método '_validate_taxes'
da 'account_invoice'

mileo committed:
[FIX] Preenchimeto dos dados da empresa corretamente.

mileo committed:
[FIX] Preenchimeto dos dados da empresa corretamente.

mileo committed:
Merge pull request #5 from kmee/8.0

[FIX] Preenchimento dos dados da empresa corretamente.

mstuttgart committed:
Ajuste PEP8

mstuttgart committed:
[MIG] Classe "StockReturnicking" migrada para v8.

mstuttgart committed:
[FIX] Corrigido bug que lançca exceção do método 'on_change_partner_id' da classe 'stock.picking' do módulo
'l10n_br_stock'

danimaribeiro commented:
Any news on the migration guys.
My first email on the Odoo-Brasil group talking about the 8.0 version was in october/2014.
It has been exactly 4 months waiting, in the meantime I started and stopped working in the migration a few times, because the emails saying the migration was coming.
I hope someone publish the code already migrated or accept our contribution, then we can move forward on this.

If we are not transparent in our actions, and how we deal with contributions, it's hard to expect contributions.

danimaribeiro committed:
[FIX] CFOP was not copying from account.fiscal.position to invoice line

danimaribeiro committed:
[FIX] Removed onchange_mask_zip from xml to avoid errors

danimaribeiro committed:
Merge pull request #6 from Trust-Code/migration/fix_cfop

[FIX] CFOP was not copying from account.fiscal.position to invoice line

mstuttgart committed:
onchange_partner_id ajustado

mstuttgart committed:
onchange_fiscal_category_id ajustado

mstuttgart committed:
Ajustes onchange_company_id

mstuttgart committed:
Ajustes onchange_product_id

mstuttgart committed:
Ajuste em_default_fiscal_category

mstuttgart committed:
Código limpo. comentarios removidos

mstuttgart committed:
[MIG] StockReturnPicking migrado

mstuttgart committed:
[MIG] Stock_invoice_onshipping parcialmente migrado

mileo committed:
[MIG][l10n_br_account] Fields and imports

mileo committed:
[MIG][l10n_br_account][l10n_br_account_product] Migration to new api

mileo committed:
[FIX] OnChange Partner city id on contact form

mileo committed:
[FIX] OnChange Partner city id on contact form

mileo committed:
Merge pull request #13 from kmee/8.0

[FIX] OnChange Partner city id on contact form

mileo committed:
[FIX] Corrigido Bug em compute_all da account_invoice do modulo l10n_br_account.
[UPDATE] Bug na instalação de módulo da localização.
Arquivos demo dos módulos l10n_br_account e l10n_be_sale_stock foram
removidos dos seus respectivos __openerp_py pois estavam causando erro de instalação.

mileo committed:
[MIG][l10n_br_stock][l10n_br_sale_stock]

mileo committed:
Merge branch 'migration/l10n_br_account' into feature/migration/stock_account

mileo committed:
[NEW][MIG][IMP] Group default fiscal category journal on invoicing from picking, fix migration issues and reactivate demo data

mileo committed:
[NEW] web_context_tunnel on onchange_partner_id

mileo committed:
Merge pull request #12 from kmee/feature/migration/8.0-rc

Feature/migration/8.0 rc

mileo committed:
Merge pull request #9 from Trust-Code/migration/fix_onchange_zip

[FIX] Removed onchange_mask_zip from xml to avoid errors

gurneyalex commented:
@mileo  I added you github login information in the OCA database

No CLA found for @Mackilem or @danimaribeiro

bertozo committed:
[FIX] Inclusão do status Denegado na NFE

mileo commented:
@gurneyalex Thanks. I Contacted @danimaribeiro and @Mackilem to sign OCA CLA.

[]s

mstuttgart committed:
[FIX] Bug onde o campo internal_number (Numero da fátura) não mostra valor, apesar de
estar salvo no banco.

mileo committed:
Merge pull request #15 from kmee/fix_campo_num_fatura_sem_valor

[FIX] Bug onde o campo internal_number (Numero da fatura) não exibe nenhum valor

mileo committed:
[NEW] Transmissão direta na NF-E de compra

mileo committed:
[FIX] Transmissão direta da NFE de compra, inserindo campos restantes

mileo committed:
Merge pull request #14 from kmee/feature/migration/8.0-rc

Feature/migration/8.0 rc

mileo committed:
[REF] Improve freight, insurance and other costs calc

mileo committed:
Merge pull request #17 from kmee/feature/refactory_sale_product_costs

[REF] Improve freight, insurance and other costs calc

mileo committed:
[REF] Fix purchase and zero division on freight

mileo committed:
Merge branch 'feature/refactory_sale_product_costs' into 8.0

mileo committed:
[REF][MIG][l10n_br_account_product] Migrate fields and computed fields to 8.0

mileo committed:
Merge pull request #18 from kmee/feature/migration/l10n_br_account_product

[REF][MIG][l10n_br_account_product] Migrate fields and computed fields t...

mileo committed:
Merge pull request #17 from kmee/8.0

[NEW] Transmissão direta da nfe de compra

danimaribeiro committed:
Melhorias na view de classificação fiscal

mileo committed:
[NEW] Assegura que a tag NCM tem no maximo 8 digitos

mileo committed:
[FIX] Duplicacao de fatura, hora no danfe e campo numero da fatura corrigido na visao

mileo commented:
up!

shingonoide commented:
The test script seem not working ok.

> $ ./.travis_test_script.sh
> /home/travis/build/OCA/openerp
> fiscal_rules  l10n-brazil  odoo_8.0.latest.tar.gz  openerp  server-env-tools
> /home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/pkg_resources/**init**.py:1222: UserWarning: /home/travis/.python-eggs is writable by group/others and vulnerable to attack when used with get_resource_filename. Consider a more secure location (set with .set_extraction_path or the PYTHON_EGG_CACHE environment variable).
>   warnings.warn(msg, UserWarning)
> Usage: openerp-server [options]
> openerp-server: error: option --addons-path: no such directory: '../l10n_br_core'
> The command "./.travis_test_script.sh" exited with 0.
> Done. Your build exited with 0.

danimaribeiro commented:
@shingonoide Yes, it's not working and also has some unit tests not passing.
Maybe would be better to change from travis to OCA runbot, I'll check on that.

mileo committed:
Merge pull request #20 from kmee/8.0

[FIX] Duplicacao de fatura, hora no danfe e campo numero da fatura co…

danimaribeiro committed:
[FIX] odoo-brazil/l10n-brazil#22 Repassa o percentual digitado no discount_rate aos itens

mileo committed:
[FIX] Removing duplicated tag, internal_number

danimaribeiro committed:
Melhorado view da empresa igual ao parceiro

danimaribeiro committed:
Fix odoo-brazil/l10n-brazil#24 issue, add new api decorator and fix call to the function

danimaribeiro committed:
Corrige dminio do campo tipo fiscal no reembolso de venda

danimaribeiro committed:
Corrigido soma de seguro e custos que estavam invertidas

mileo commented:
In the KMEE we recognize the investment Akretion and always try to be correct when we send a pull request or suggestion.

In late 2014 https://github.com/OCA/l10n-brazil/pull/109 request the creation of the Branch 8.0 but the pull request was closed by @renatonlima. Surely the best way to realize the pull request was module by module. But with the creation of the empty branch not allowed, it is impossible...

So the solution was to carry out the migration process in a fork https://github.com/odoo-brazil/l10n-brazil and perform a pull request from the 8.0 branch only to record our work.

Yesterday I spoke with the contributors pull request https://github.com/OCA/l10n-brazil/pull/124 and anyone was pleased with these news PR of Akretion. It looks like that our work has no importance because of this tyranny.

If the location is in the OCA arms, great, guarantees more security to customers, but our pull request has been done before and should be performed before the merge. We will be happy to correct any potential problems and if necessary perform pull separate requests.

I'm sure also that since we started to contribute in late 2011 we make mistakes and we are proud to have learned from them.

I hope that the OCA can evaluate our position in order to prevent people stop contributing to the official branch.

Best Regards

renatonlima commented on l10n_br_account/res_partner.py:
```diff
@@ -17,43 +17,40 @@
+    asset_operation = fields.Boolean(string=u'Operação de Aquisição de Ativo',
+        help=u"""Caso seja marcada essa opção, será incluido o IPI na base de
+            calculo do ICMS.""")
+    id_dest = fields.Selection([('1', u'Operação interna'),
```
This filed id_dest should be in l10n_br_account.product.cfop object, I did this in https://github.com/akretion/l10n-brazil/blob/migration/l10n_br_account_product/l10n_br_account_product/models/l10n_br_account_product.py#L53

danimaribeiro committed:
Corrige erro ao criar devolução de compra

danimaribeiro committed:
Adicionado informação de referencia de compra ao xml da nf-e

danimaribeiro committed:
Documentos referenciados e finalidade de devolução adicionados a nova fatura ao efetuar reembolso

mstuttgart committed:
[FIX] Corrigido erro na obtencao do valor (vDup) da duplicata

mileo committed:
[FIX] Data de contabilizacao confonforme a data de entrada e saida da fatura

mileo committed:
[FIX] NCM duplicado

mileo committed:
[FIX] Dominio do endereco de entrega da fatura e UF do in_out do serializer em document.py

ananiasfilho committed:
[FIX] sale_analysis_report

mileo committed:
[FIX] name_get of l10n_br_account.invoice.invalid.number

mileo committed:
[FIX] Add string limit to xProd and xNome

mstuttgart committed:
[FIX] Tipo de frete não depende de existir uma transportadora selecionada no campo "carrier_id".

mileo committed:
Merge branch 'feature/nfe_denegada' of https://github.com/kmee/l10n-brazil into kmee-feature/nfe_denegada

Conflicts:
	l10n_br_account_product/account_invoice.py

mileo committed:
Merge branch 'kmee-feature/nfe_denegada' into 8.0

mileo committed:
Merge pull request #18 from Trust-Code/feature/melhoriaview_regras

Melhorias na view de classificação fiscal

alexsandrohaag committed:
Ajustado import da validação da inscrição estadual

Ajustada cláusula import da validação da inscrição estadual.

mileo committed:
Merge pull request #28 from Trust-Code/feature/improve_view_company

Melhorado view da empresa igual ao parceiro

mileo committed:
Merge pull request #41 from odoo-brazil/hangout-fix-entrega

[FIX] Dominio do endereco de entrega da fatura e UF do in_out do seri…

mileo committed:
Merge pull request #38 from kmee/feature/sale_report

[FIX] sale_analysis_report

mileo committed:
Merge pull request #39 from odoo-brazil/hangout-fix_ncm

[FIX] NCM duplicado

danimaribeiro committed:
Merge pull request #43 from hgsoft/8.0

Ajustado import da validação da inscrição estadual

mileo committed:
Merge pull request #32 from Trust-Code/fix/desconto_geral

[FIX] #22 Repassa o percentual digitado no discount_rate aos itens

mileo committed:
Merge pull request #33 from odoo-brazil/danimaribeiro-patch-1

Corrigido soma de seguro e custos que estavam invertidas

mileo committed:
Merge branch 'fix/onchange_category_fiscal' of https://github.com/Trust-Code/l10n-brazil into Trust-Code-fix/onchange_category_fiscal

mileo committed:
[FIX] Correcao do on change da sale order

mileo committed:
Merge pull request #44 from kmee/fix_tipo_frete_nao_atualizado

[FIX] Tipo de frete não depende de existir uma transportadora selecio…

mileo committed:
Merge pull request #40 from odoo-brazil/hangout-fix-date

[FIX] Data de contabilizacao confonforme a data de entrada e saida da…

mileo committed:
[REF] Remove o campo name da inutilizacao

mileo committed:
Merge pull request #42 from odoo-brazil/fix-nfe-invalid-number

[FIX] name_get of l10n_br_account.invoice.invalid.number

mileo committed:
[MIG] Migration to new api

mileo committed:
Merge pull request #45 from odoo-brazil/Trust-Code-fix/onchange_category_fiscal

Trust code fix/onchange category fiscal

mileo committed:
Merge pull request #46 from odoo-brazil/fix-xProd-xNome-limits

[FIX] Add string limit to xProd and xNome

mileo committed:
Merge pull request #47 from odoo-brazil/fix/vDup

[FIX] Corrigido erro na obtencao do valor (vDup) da duplicata

mileo committed:
Merge branch '8.0' into fix/ncm_8_digits

Conflicts:
	l10n_br_account_product/sped/nfe/document.py

mileo committed:
Merge pull request #48 from odoo-brazil/fix/ncm_8_digits

[NEW] Assegura que a tag NCM tem no maximo 8 digitos

alexsandrohaag committed:
Retirado teste "not self.is_company" da IE

Retirado teste "not self.is_company" da validação da Inscrição Estadual, pois o modelo crm.lead não possui este campo.

mileo committed:
[FIX] Removendo cascade

mileo committed:
Merge pull request #51 from odoo-brazil/fix/cascade_eletronic_document

[FIX] Removendo cascade

sadamo committed:
[IMP] added date_maturity in invoice page "Vencimentos"

sadamo committed:
[FIX] inserido o invoice_validate no workflow da Invoice, pois ele e utilizado pelo Sale e Purchase

danimaribeiro committed:
Merge pull request #49 from alexsandrohaag/8.0

Retirado teste "not self.is_company" da IE

danimaribeiro committed:
Correção de digitação e melhoria da view de veículos

danimaribeiro committed:
Acentos e descrição da coluna tipo errada

sadamo committed:
[FIX] correcao dos metodos de aquisicao de datas action_date_assign

Conflicts:
	l10n_br_account_product/account_invoice.py

danimaribeiro committed:
Usando infraestrutura de build da OCA e atualizado badges

danimaribeiro committed:
Merge pull request #63 from Trust-Code/fix/change_build_infra

Usando infraestrutura de build da OCA e atualizado badges

danimaribeiro committed:
[FIX] odoo-brazil#7 Correção de onchange do cnpj cpf, correção de constraint no tipo fiscal do parceiro

danimaribeiro committed:
Migração do módulo l10n_br_zip para nova api

yk2kus committed:
FIX: ValueError: can't multiply sequence by non-int of type 'float' while evaluating u'nfe_export()'

Commit:
Merge branch '8.0' of github.com:odoo-brazil/l10n-brazil into development

danimaribeiro committed:
Merge pull request #66 from thinkopensolutions/development

FIX: ValueError:

mstuttgart committed:
[MIG] Migrate l10n_br_crm_zip to new api.

danimaribeiro committed:
Merge pull request #64 from Trust-Code/fix/onchange_cpf_cnpj

[FIX] odoo-brazil#7 Correção de onchange do cnpj cpf, correção de con…

danimaribeiro committed:
Limite de tamanho nos campos nome de parceiro e produto

mileo committed:
Update README.md

mileo committed:
Merge branch 'fix/dominio_devolucao_venda' of git://github.com/Trust-Code/l10n-brazil into Trust-Code-fix/dominio_devolucao_venda

mileo committed:
[FIX] l10n_br_account_product refund on_change_partner_id arguments

mileo committed:
[REF] Refactory refund

danimaribeiro committed:
Merge pull request #73 from odoo-brazil/Trust-Code-fix/dominio_devolucao_venda

Trust code fix/dominio devolucao venda

danimaribeiro committed:
Merge pull request #65 from Trust-Code/migration/modulo_zip

Migração do módulo l10n_br_zip para nova api

danimaribeiro committed:
Merge pull request #70 from mstuttgart/migration/l10n_br_crm_zip

[MIG] Migração do módulo l10n_br_crm_zip para nova api

danimaribeiro committed:
Change long name for a short one

danimaribeiro committed:
Adding decorator from new api to onchange fields

danimaribeiro committed:
Fixed the number of tax_id to 5, used the default fiscal_position if nothing found

danimaribeiro committed:
Fixing _default_fiscal_cetegory - creating a field in res_company to use as default

danimaribeiro committed:
Creating default fiscal category to company in demo data and adding fiscal position e fiscal category fields to sales order line treeview

danimaribeiro committed:
Changed location of default fiscal category

danimaribeiro committed:
Fixing default fiscal category for stock.picking

renatonlima committed:
Created procurement order and copy of fiscal category and position

danimaribeiro committed:
Fix stock.picking directly from sale order

danimaribeiro committed:
Unit tests for the zip search

mileo committed:
Merge pull request #74 from Trust-Code/fixing/tests

Fixing unit tests

danimaribeiro committed:
New zip tests for multiple results

danimaribeiro committed:
Unit tests to zip search in crm.lead, changed button order in zip result window

danimaribeiro committed:
Merge pull request #71 from Trust-Code/feature/change_size_fields

Limite de tamanho nos campos nome de parceiro e produto

danimaribeiro committed:
Validate cpf and cnpj only when the country is Brasil

danimaribeiro committed:
Adding country_id field to the constraint

danimaribeiro committed:
Merge pull request #78 from Trust-Code/fix/cliente-estrangeiro

Validate cpf and cnpj only when the country is Brasil

mstuttgart committed:
Merge pull request #83 from kmee/feature/date_maturity_and_date_assign

Feature/date maturity and date assign

mstuttgart committed:
[FIX] Removido espaço em branco no começo do nome da rua quando o campo street_type esta vazio

danimaribeiro committed:
Transformando string para unicode.

danimaribeiro committed:
Merge pull request #87 from mstuttgart/fix/l10n_br_zip_street_adjust

[FIX] l10n_br_zip. - Removido espaço em branco no começo do nome da rua

Commit:
FIX: division by zero in l10n_br_account_invoice_costs_ratio wizard

yk2kus committed:
FIX:  NameError: global name 'ammount_total' is not defined

danimaribeiro committed:
Merge pull request #53 from Trust-Code/fix/veiculo_view

Correção de digitação e melhoria da view de veículos

danimaribeiro committed:
Merge pull request #56 from odoo-brazil/fix/wokflow_SO_PO

[8.0][FIX] Correção do workflow da purchase order e sale order

danimaribeiro committed:
Merge pull request #62 from Trust-Code/feature/novo_tipo_devolucao

Documentos referenciados e finalidade de devolução adicionados a nova…

danimaribeiro committed:
Merge pull request #75 from Trust-Code/feature/unit-test-zip

Unit tests for the zip search

danimaribeiro committed:
Merge pull request #77 from Trust-Code/feature/unit-test-crm-zip

Feature/unit test crm zip

Commit:
Removed print debug lines

mileo committed:
Merge pull request #61 from Trust-Code/feature/informacoes_compra_xml

Adicionado informação de referencia de compra ao xml da nf-e

danimaribeiro committed:
Adicionando badge da cobertura de testes.

danimaribeiro committed:
Merge pull request #110 from thinkopensolutions/fix/division_by_zero_in_l10n_br_account_invoice_costs_ratio_wizard

Fix/division by zero in l10n br account invoice costs ratio wizard

rvalyi commented:
superseded by ongoing PR
